controllers/controllers/services/bindings/controller.go

```diff
@@ -46,15 +43,15 @@ const (
 )
 
 type CredentialsReconciler interface {
```

Comment:
I wonder whether the name `CredentialsReconciler` makes sense for this interface anymore. This interface is not dealing with credentials only, it is implementing the binding-type-specific bits from the binding reconciliation process. Maybe `DelegateReconicler` could be an appropriate name as the top-level reconciler delegates to one of the "smaller" ones?